.github/workflows/coverage.yaml

```diff
@@ -2,6 +2,9 @@ name: coverage
 
 on: [push]
 
+permissions:
+  contents: read
```

Comment:
## GITHUB_TOKEN Permissions
### Original permissions in non paranoid repositories
```yaml
  Actions: write
  Checks: write
  Contents: write
  Deployments: write
  Discussions: write
  Issues: write
  Metadata: read
  Packages: write
  Pages: write
  PullRequests: write
  RepositoryProjects: write
  SecurityEvents: write
  Statuses: write
```

### Original permissions in paranoid repositories
```yaml
  Contents: read
  Metadata: read
  Packages: read
```

### Reduced Permissions
```yaml
  Contents: read
  Metadata: read
```